doc/design/session.md

```diff
@@ -0,0 +1,62 @@
+# Design Doc: Session
+
+## Abstract
+
+This design doc proposes to have an object called *Session* which
```

Comment:
Maybe the *Session* try to solve the problem of 

1. unify the runtime sources management between a local machine and distributed environment.
2. Replace the *global scope* with a named concept, which holds resources explicitly. 

is it correct ?

Reply:
Yes.
Updated, please review:)